### DISCUSSION TITLE
Change pole color

### DISCUSSION THREAD

**Discussion by bisiyo** [iTowns/itowns#1640] [Q&A] [answered]
Hello, I contact you to know if there is a way to change the color of the pole?

![image](https://user-images.githubusercontent.com/27958237/114469875-3645d500-9bee-11eb-8b76-7ef8b1378af4.png)

![image](https://user-images.githubusercontent.com/27958237/114470066-86bd3280-9bee-11eb-94d3-a6d224474feb.png)

Thank you

**Comment by gchoqueux** [comment]
There's not option to change pole color. We'll add in next release .
But there's a tip change the color, with `GeometryLayer` adding:

```js
const poleColor = 0xff0000;

var poleColorLayer = new itowns.GeometryLayer('color pole', new itowns.THREE.Group(), {
    update: (context, layer, node) => {
        if (node.visible) {
            node.material.diffuse.set(poleColor);
        }
    },
    convert: () => {},
    source: false,
    zoom: { min: 1 },
});

view.addLayer(poleColorLayer);
```

**Comment by bisiyo** [comment]
thank you !! it works perfectly 

![image](https://user-images.githubusercontent.com/27958237/114540266-991e8700-9c55-11eb-894b-d2d2fea24761.png)

**Comment by gchoqueux** [comment, marked as answer]
Sorry, i forget there's this `diffuse` option 

```js            
const view = new itowns.GlobeView(viewerDiv, placement, {
    diffuse: new itowns.THREE.Color(0xff0000)
});
```

**Comment by gchoqueux** [comment]
i add a documentation  #1632
